test/data/ops/matmulpack.jsonc

```diff
@@ -0,0 +1,31 @@
+[
```

Comment:
nit: consider rename it to matmul_pack.jsonc for consistancy

Reply:
done.